wire-ios-system/Source/Logging/WireLogger.swift

```diff
@@ -16,12 +16,18 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-public struct WireLogger: LoggerProtocol {
+public struct WireLogger: LoggerProtocol, Sendable {
 
-    private static var provider = AggregatedLogger(loggers: [
-        SystemLogger(),
-        CocoaLumberjackLogger()
-    ])
+    public static func initialize(loggers: [any LoggerProtocol]) {
+        guard provider == nil else {
+            assertionFailure("WireLogger.initialize called more than once")
+            return
+        }
+
+        provider = AggregatedLogger(loggers: loggers)
+    }
+
+    private static nonisolated(unsafe) var provider: (any LoggerProtocol)!
```

Comment:
The `provider` value is now injected via `initialize(loggers:)`.
This func should only be called once.
Still, I don't consider it necessary to use any custom logic to prevent. So I put a simple `guard` + `assertionFailure` there.

Currently this code is called by `WireAnalytics.setup()` which is called in
`UIApplicationDelegate.willFinishLaunchingWithOptions` and in the two app extensions.